catalogue.py

```diff
@@ -477,10 +477,11 @@ def set_key_fields(self, indices=None, cat=None, set_coords=True):
                 self.rms[cat.name] = cat.rms[cat.name]
 
                 # write flux ratio if frequencies within 1%
-                if self.name in list(self.flux.keys()) and np.abs(cat.freq[cat.name]/self.freq[self.name]-1) < 0.01:
+                if self.name in list(self.flux.keys()) and\
+                 np.abs(cat.freq[cat.name]/self.freq[self.name]-1) < 0.01:
```

Comment:
I would change this to something like:
```
freq_ratio = np.abs(cat.freq[cat.name]/self.freq[self.name]-1)
     if self.name in list(self.flux.keys()) and freq_ratio < 0.01:
```

Its a bit easier to read and also flake8 compliant

Reply:
Done.